rbac/server/api/roles.py

```diff
@@ -20,7 +20,7 @@
 
 from rbac.common import rbac
 
-from rbac.server.api.errors import ApiNotImplemented
+# from rbac.server.api.errors import ApiNotImplemented
```

Comment:
Please delete all commented out lines in this file having to do with API not implemented.  If they are necessary in the future we will implement from scratch

Reply:
Gotcha!